app/views/vary/schemeVariationsSuccess.scala.html

```diff
@@ -0,0 +1,46 @@
+@*
```

Comment:
same as bove,
please move it to normal package,  nothing should go inside vary

Reply:
Are we not using vary package? If so, we should remove the one item that is currently in that package

Reply:
Done with package changes. please verify and merge